src/components/DonutChart.js

```diff
@@ -62,6 +64,7 @@ class DonutChart extends React.Component {
     padAngle: 0.02,
     showBaseArc: true,
     showDataLabel: true,
+    shouldToggleDataLabelOnHover: true,
```

Comment:
By defaulting this to true, you are introducing unexpected behavior for users of this library. It would probably be better to default to false and then set it to true in your usage of it.

Reply:
Scratch this comment. I just read the code below. Defaulting to true preserves the existing behavior. Disregard this comment. 👍

Reply:
True was the default behavior of the chart before this implementation. In our usage, we want to set it to false so there is no hover effect changing the text inside the chart.